src/io/initial_coral_cover.jl

```diff
@@ -0,0 +1,96 @@
+using Distributions,
+    NetCDF,
+    YAXArrays
+
+"""
+    load_cover(data_fn::String)::YAXArray
+    load_cover(n_species::Int64, n_sites::Int64)::YAXArray
+
+Load initial coral cover data from netCDF.
+"""
+function load_cover(data_fn::String)::YAXArray
```

Comment:
ReefmodDomain and RMEDomain implement a function called `load_initial_cover`. Should the naming be the same as the others?
```suggestion
function load_initial_cover(data_fn::String)::YAXArray
````
I'm happy with either naming

Reply:
Yes, best to keep it inconsistent

Reply:
I was actually thinking about doing this change but for another reason, so I'm happy to change.